app/commands/user/reset_cache.rb

```diff
@@ -24,4 +27,11 @@ def cache
   def value_for_has_unrevealed_testimonials? = user.mentor_testimonials.unrevealed.exists?
   def value_for_has_unrevealed_badges? = user.acquired_badges.unrevealed.exists?
   def value_for_has_unseen_reputation_tokens? = user.reputation_tokens.unseen.exists?
+  def value_for_num_solutions_mentored = user.mentor_discussions.count
```

Comment:
There already is a `num_solutions_mentored` field on `User::Data`. How does this relate to that field?